executor/runtime/docker/docker.go

```diff
@@ -638,6 +655,26 @@ func vpcToolPath() string {
 	return ret
 }
 
+// Use image labels to determine if the container should be configured to run SystemD
+func setSystemdRunning(imageInfo types.ImageInspect, c *runtimeTypes.Container) error {
+	l := log.WithField("imageName", c.QualifiedImageName())
+
```

Comment:
Why not log the value here, rather than below, so if the value is "corrupted" we can at least figure out what's going on?

Reply:
Also, this logger should descend from the one already in `Prepare`